diff -Nru encodings-0.99.0.orig/Makefile.am encodings-0.99.0/Makefile.am
--- encodings-0.99.0.orig/Makefile.am	2005-07-12 23:03:09 +0200
+++ encodings-0.99.0/Makefile.am	2005-08-14 23:47:10 +0200
@@ -1,6 +1,6 @@
 SUBDIRS = large
 
-encodingsdir = $(libdir)/X11/fonts/encodings
+encodingsdir = $(datadir)/fonts/encodings
 
 ENCODING_FILES =		\
 	adobe-dingbats.enc	\
diff -Nru encodings-0.99.0.orig/large/Makefile.am encodings-0.99.0/large/Makefile.am
--- encodings-0.99.0.orig/large/Makefile.am	2005-07-12 23:03:09 +0200
+++ encodings-0.99.0/large/Makefile.am	2005-08-14 23:47:21 +0200
@@ -1,4 +1,4 @@
-encodingsdir = $(libdir)/X11/fonts/encodings/large
+encodingsdir = $(datadir)/fonts/encodings/large
 
 ENCODING_FILES =		\
 	big5.eten-0.enc		\
